internal/service/cloud/ipblock_test.go

```diff
@@ -189,6 +191,12 @@ func (s *ipBlockTestSuite) validateRequest(method, state, path string, req *infr
 	s.Equal(path, req.RequestPath)
 }
 
+func (s *ipBlockTestSuite) validateSuccessfulReservation(location string, err error) {
+	s.T().Helper()
+	s.NoError(err)
+	s.Equal(exampleLocation, location)
+}
```

Comment:
Right now this is mostly overhead: You use it only once and there the error was even checked before calling this method.
So rather just move the `s.Equal(exampleLocation, location)` check where this function is called.